react/Gallery.js

```diff
@@ -48,9 +48,14 @@ class Gallery extends Component {
     )
   }
 
+  get gapItems() {
```

Comment:
```suggestion
  get paddingForItem() {
```

`gapItems` possibly means it will return a series of gap items